docs/advanced/how-to-build-an-oracle-service.md

```diff
@@ -0,0 +1,155 @@
+---
+sidebar_position: 11
+---
+
+# How to Build an Oracle Service
+
+As described in [this tutorial](../tutorials/oracle.md), a blockchain oracle is a third-party service or agent that provides external data to a blockchain network. It is a bridge between the blockchain and the external world, enabling smart contracts to access, verify, and incorporate data from outside the blockchain. Specifically, the oracle service provides external data along with a [Rabin signature](https://en.wikipedia.org/wiki/Rabin_signature_algorithm) of the data, and the smart contract uses this data and verifies the signature before using it.
```

Comment:
Explain briefly why using Rabin: https://docs.scrypt.io/tutorials/oracle#rabin-signatures.

Afterwards, let's remove every mention of Rabin. Just say signature.